polyfills/TypedArray/prototype/@@toStringTag/polyfill.js

```diff
@@ -0,0 +1,70 @@
+/* global Symbol, Type */
+// 23.2.3.33 get %TypedArray%.prototype [ @@toStringTag ]
+(function () {
+	function _get() {
+		// 1. Let O be the this value.
+		var O = this;
+		// 2. If Type(O) is not Object, return undefined.
+		if (Type(O) !== 'object') {
+			return undefined;
+		}
+		// 3. If O does not have a [[TypedArrayName]] internal slot, return undefined.
+		if (!('_name' in O)) {
+			return undefined;
+		}
+		// 4. Let name be O.[[TypedArrayName]].
+		var name = O._name;
+		// 5. Assert: Type(name) is String.
+		if (Type(name) !== 'string') {
+			throw TypeError();
+		}
+		// 6. Return name.
+		return name;
+	}
+
+	_get.name = 'get [Symbol.toStringTag]'
```

Comment:
```suggestion
	Object.defineProperty(_get, 'name', {
		value: 'get [Symbol.toStringTag]',
		writable: false,
		enumerable: false,
		configurable: true
	});
```

I think this will solve the CI failures

Reply:
It's not always possible. Old engines (IIRC FF <38, Chrome <43, Safari <10) functions `.name` is non-writable non-configurable.

Reply:
@zloirock Is there a better way to set `.name` or should we wrap `Object.defineProperty` in `try/catch` and remove the failing test?

Reply:
Nope, I don't know a better way. I use try / catch and `defineProperty` and test it only when it's possible to define.

Reply:
Thank you for this feedback and keeping an eye on all these changes @zloirock :)

@mhassan1 Do you have enough info to get this working?